lib/npm-auditer.js

```diff
@@ -69,13 +69,20 @@ function reportAudit(npmAudit, config) {
 
 function runNpmAudit(callback) {
   childProcess.exec('npm audit --json', (_error, stdout, stderr) => {
-    if (stderr) {
-      callback(new Error(stderr), null);
-      return;
+    if (stdout) {
+      try {
+        const parsedAudit = JSON.parse(stdout);
+        callback(null, parsedAudit);
+        return;
+      } catch (error) {
+        callback(new Error('Invalid output'), null);
+        return;
+      }
     }
 
-    const parsedAudit = JSON.parse(stdout);
-    callback(null, parsedAudit);
+    if (_error) {
```

Comment:
I'd guess that this variable should be named just `error` now that it's being used.
This is just nit-picking though; fine to merge as-is.

Reply:
I didn't want to be so invasive with the PR, but I agree with it